### DISCUSSION TITLE
My Vercel Hobby plan account got paused

### DISCUSSION THREAD

**Discussion by skgupta507** [vercel/community#7427] [Account Issue] [answered]
### Question

Hello Guys, 
My vercel hobby plan account got paused 1month ago. Due to reaching its Limits.  But even after a month the account is still paused doesn't the limits get reset every 30 days. 
How can i get it unpaused?

**Comment by amyegan** [comment]
Hi @skgupta507. I'm not able to find an account using your GitHub username. 

I'll need to know more about your account and project(s) that caused high usage. If you can share a deployment link or ID then I can look into it for you.

**Comment by skgupta507** [comment]
Username- skgupta507

Hope u can find it from this info. 
I have removed the deployed domain from the project so i can't share the project link. 
aniplay-5onwrfkj4-sunils-projects-4d721537.vercel.app
Here's the deployment link

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing that info. Your Hobby team is no longer paused

**Reply by skgupta507** [reply]
Thanks @amyegan ☺
Now I can deploy my hobby projects again 👍😇☺

[a comment by an automated account is not shown]
